Phonebook.Frontend/src/app/modules/table/components/table/table.component.ts

```diff
@@ -97,7 +114,7 @@ export class TableComponent implements OnInit, OnDestroy {
       .refresh(
         this.store.selectSnapshot(SearchState.searchTerm).trim(),
         this.store.selectSnapshot(SearchState.searchFilters),
-        this.store.selectSnapshot(TableState.visibleColumns),
+        this.store.selectSnapshot(TableState.visibleBigColumns),
```

Comment:
Shouldn't this depend on the current size?

Reply:
Yes. Weirdly, the refresh is working as it should. Even when I change the visibleSmallColumns and then reload the table, it refreshes correctly. Do you have an idea as to how to still get the right snapshot without putting the entire function into an if-statement?